packages/ddp-client/client/queueStubsHelpers.js

```diff
@@ -95,13 +95,14 @@ export const loadAsyncStubHelpers = () => {
           const applyAsyncPromise = oldApplyAsync.apply(this, args);
           stubPromiseResolver(applyAsyncPromise.stubPromise);
           serverPromiseResolver(applyAsyncPromise.serverPromise);
+          applyAsyncPromise.stubPromise.finally(() => {
```

Comment:
I think there are cases where we don't have `stubPromise`. It's best to do something like `applyAsyncPromise.stubPromise?.finally` here.

Reply:
Added a pre-check of existence.